### DISCUSSION TITLE
Is there a way to change the display name of an input?

### DISCUSSION THREAD

**Discussion by lifenautjoe** [react-hook-form/react-hook-form#2101] [Q&A]
My error messages look like this 
<img width="303" alt="image" src="https://user-images.githubusercontent.com/12737509/86746651-7e224500-c03b-11ea-88e0-bcd717d96315.png">

Is there any way to specify a different displayName rather than using the input name to display error messages?

**Comment by bluebill1049** [comment]
because MUI use the `name`?

**Comment by lifenautjoe** [comment]
The error message us obtained from react-form-hook ‘errors’ as
errors.fieldName.message and it contains the field name in the string as
seen on the screen capture

**Reply by bluebill1049** [reply]
you can customize the error message then.

**Comment by lifenautjoe** [comment]
Yes but then it would defeat one of the big pros of having integration with
validators such as Joi which have extensive error messages built in.

I would have to write a custom error for all kinds of validations again.

Is it correct to assume this functionality is not there? To provide a name
and displayName attribute to be used in out-of-the-box error messages ?
